tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -800,17 +825,27 @@ func getIngressOrFail(t *testing.T) string {
 // TestCheckCache tests that check cache works within the mesh.
 func TestCheckCache(t *testing.T) {
 	// Get pod id of sleep app.
-	pod, err := podID("app=sleep")
+	pod, err := podID("app=sleep", false)
```

Comment:
something that might be helpful here for readability: either:

```
func remotePodID(selector string) (string, error) {
    return podID(selector, true)
}

func localPodID(selector string) (string, error) {
    return podID(selector, false)
}
```

or

```
const (
    remote = true
    local = false
)
...
pod, err := podID("app=sleep", local)
```

Reply:
thanks fo rthe review.  
Ok - I will improve this,  but waiting for https://github.com/istio/istio/pull/8536 before I do any fix up to this.